dbt/adapters/bigquery/python_submissions.py

```diff
@@ -118,16 +120,27 @@ def _get_job_client(self) -> dataproc_v1.BatchControllerClient:
     def _submit_dataproc_job(self) -> dataproc_v1.types.jobs.Job:
         batch = self._configure_batch()
         parent = f"projects/{self.credential.execution_project}/locations/{self.credential.dataproc_region}"
+        batch_id = uuid.uuid4().hex
 
         request = dataproc_v1.CreateBatchRequest(
             parent=parent,
             batch=batch,
+            batch_id=batch_id
         )
         # make the request
         operation = self.job_client.create_batch(request=request)  # type: ignore
         # this takes quite a while, waiting on GCP response to resolve
         # (not a google-api-core issue, more likely a dataproc serverless issue)
-        response = operation.result(polling=self.result_polling_policy)
+
+        state = "PENDING"
```

Comment:
the linting checks are failing with the following message
> `python_submissions.py:127:9`: `F841` local variable `'operation'` is assigned to but never used

I suggest either:
1. removing it, or
2. (preferred) use it to set the initial `state` variable

Reply:
I took the easier path 🔢